api/src/controllers/profile.ts

```diff
@@ -94,7 +94,7 @@ export const updateProfileContacts = async (
     user,
   }: { params: any; body: Contact[]; user: AuthenticatedUser },
   res: Response
-): Promise<void> => {
+): Promise<Response> => {
```

Comment:
I like the typing idea you have with this change (and the equivalent change in `request.ts`), but it looks like it's [breaking the build](https://github.com/bcgov/platform-services-registry/runs/8215361327?check_suite_focus=true).

With this change, the TypeScript compiler is giving me this error: `Type 'void' is not assignable to type 'Response<any>'.` It looks like this is due to the `Response` method we are calling: `.end()`. I actually had to look that one up - it's not something I've ever used in all my years of using Express. See [`res.end()`](https://expressjs.com/en/api.html#res.end) vs [`res.send()`](http://expressjs.com/en/4x/api.html#res.send).

I'm not enough of a TypeScript wizard to know what's the best way to go about solving this, but I see this [StackOverflow answer](https://stackoverflow.com/a/56080762) points to a reason for why this happens with `.end()`.

I notice if I naively change all the `.end()` calls to `.send()`, the TypeScript compiler is happier and I can run a build successfully. However, it's not safe to just make this change, as it causes new test breakages in `profile.spec.ts`.